messaging/kafka/plugin_impl_kafka.go

```diff
@@ -99,8 +99,12 @@ func (p *Plugin) Init() (err error) {
 // AfterInit is called in the second phase of the initialization. The kafka multiplexerNewWatcher
 // is started, all consumers have to be subscribed until this phase.
 func (p *Plugin) AfterInit() error {
-	if p.muxHash == nil {
-		return nil
+	var wasError error
+	if p.muxHash != nil {
+		wasError = p.muxHash.Start()
+	}
+	if p.muxManual != nil {
+		wasError = p.muxManual.Start()
```

Comment:
if wasError = p.muxHash.Start() returned error and p.muxManual not the it will be not propagated...